plasmapy/particles/ionization_state.py

```diff
@@ -35,7 +35,7 @@ class IonicFraction:
     ion: `~plasmapy.particles.particle_class.ParticleLike`
         The ion for the corresponding ionic fraction.
 
-    ionic_fraction: real number between 0 and 1, optional
+    ionic_level: real number between 0 and 1, optional
```

Comment:
For the attribute, I'd keep the name as `ionic_fraction` since it's the fraction of the element/isotope that's at this ionic level.